### ISSUE TITLE
GRAV : As soon as you add more than 1 category, particles break

### ISSUE THREAD

**Issue by plugngeek2** [gantry/gantry5#2361]
If I use a particle with the possibility to choose categories to display, the particle can't display anything if they website has more than 1 category. It is like that with ALL particles i tried (rockettheme, inspiretheme)

**Comment by plugngeek2** [comment]
The bug in video : https://www.youtube.com/watch?v=iKUygyIDqJg  (same with all display categories particles)

**Comment by plugngeek2** [comment]
Exactly. We have a blank particle if we use more than 1 category

**Comment by simmonsr** [comment]
Confirmed

**Comment by simmonsr** [comment]
Currently the Content Array and other Grav Content Sourced particles require that if you set the particle to grab 2 categories, that article(s) must contain BOTH categories. For example, if you set the particle to show pages that have categories 'Blog' and 'Featured', the page 'How to Grow Flowers' must contain BOTH categories 'Blog' and 'Featured' for it to show.

**Comment by simmonsr** [comment]
Nevertheless, the Grav team is looking to address this limitation in Grav 1.6+, so we hope to provide in the near future that when you set particles with 'Blog' and 'Featured' categories, it would show pages that contain 'Blog' OR 'Featured'.

**Comment by mahagr** [comment]
I put this to `Future Releases` as it needs support from Grav first.

**Comment by drnasin** [comment]
Took me way too long today to find out this is a grav limitation..is there any way to change the operand via twig so i can at least put it as an option in particle yaml...

**Comment by simmonsr** [comment]
I don't believe so at this point. It is limited to articles with Cat1 AND Cat2 rather than articles with Cat1 OR Cat2.

**Comment by beamaria** [comment]
two years are nearly passed, but the problem is still unsolved... Any news?

**Comment by mahagr** [comment]
It is not a limitation of Gantry, it is a limitation of Grav. This issue is just here to make sure we fix it **IF** it gets fixed in Grav first.

**Comment by beamaria** [comment]
Thanks Mathias!
